main.star

```diff
@@ -97,7 +73,22 @@ def run(
                     persistent_key="kurtosis-indexer-data-dir"
                 ),
             },
-            env_vars=indexer_env_vars,
+            env_vars= {
```

Comment:
I refactored the env vars, so that they're all shown here. Initially, I was having a hard time reasoning about the indexer's behavior because the env vars were set in different places.